test/0_staking.js

```diff
@@ -46,34 +78,44 @@ describe('Candidates make stakes on themselves', () => {
     it('Candidates add pools for themselves', async () => {
         let stakeBN = minStakeBN.clone();
         console.log('**** stake = ' + stakeBN.toString());
-        for (candidate of constants.CANDIDATES) {
+        for (const candidate of constants.CANDIDATES) {
             await canStakeAndWithdraw(web3);
             console.log('**** candidate =', JSON.stringify(candidate));
-            let iStake = await ValidatorSetContract.instance.methods.stakeAmount(candidate.staking, candidate.staking).call();
-            let iStakeBN = new BN(iStake.toString());
-            let tx = await SnS(web3, {
+            const iStake = await SnS(web3, {
```

Comment:
`stakeAmount` is a getter, it should be `call()`ed, so there is no need to prepare and send tx.
(Also in `Candidates make stakes on themselves` test)

Reply:
Fixed